### PR TITLE
DM-17546: Update script queue for ts_salobj 3.8

### PR DESCRIPTION
**Pull request by r-owen**

### CODE REVIEW

File: python/lsst/ts/scriptqueue/script_queue.py [hunk 1]
```diff
@@ -24,6 +24,8 @@
 import asyncio
 import os.path
 
+import numpy as np
```
Comment (tribeiro) [inline python/lsst/ts/scriptqueue/script_queue.py]: I was a bit surprised about the use of `numpy` here but indeed it makes it much easier to accomplish what you want in `put_queue`. Should we worry about adding `numpy` as a dependency? Do you need to add `setupRequired(numpy)` to `ups/ts_scriptqueue.table`?

Comment (r-owen) [reply python/lsst/ts/scriptqueue/script_queue.py]: If this were a DM package then indeed I would have to specify numpy as a dependency. But even the DM stack includes numpy in its miniconda Python, so I am inclined to assume that any python installation we use will include numpy.

Comment (tribeiro) [reply python/lsst/ts/scriptqueue/script_queue.py]: makes sense... I don't know why there's a `setupRequired(numpy)` on the code for the feature scheduler... https://github.com/lsst/sims_featureScheduler/blob/d4d04f97b3087b364695141727aca5e39d3fe41a/ups/sims_featureScheduler.table#L2

Comment (couger01) [reply python/lsst/ts/scriptqueue/script_queue.py]: You could have a requirement.txt that specifies any particular dependencies. Just so those are clear.

**Pull request merged**